ArduCopter/Copter.cpp

```diff
@@ -389,7 +389,59 @@ bool Copter::set_target_angle_and_climbrate(float roll_deg, float pitch_deg, flo
     mode_guided.set_angle(q, Vector3f{}, climb_rate_ms*100, false);
     return true;
 }
-#endif
+
+bool Copter::set_target_rate_and_throttle(float roll_rate_dps, float pitch_rate_dps, float yaw_rate_dps, float throttle)
```

Comment:
let's put a comment above this method.  It's pretty obvious what it does but all other methods have comments so let's keep it consistent

Reply:
Done in https://github.com/ArduPilot/ardupilot/pull/28134